CHANGELOG.md

```diff
@@ -45,6 +45,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Migrate client transports to Apache HttpClient / Core 5.x ([#4459](https://github.com/opensearch-project/OpenSearch/pull/4459))
 - Change http code on create index API with bad input raising NotXContentException from 500 to 400 ([#4773](https://github.com/opensearch-project/OpenSearch/pull/4773))
 - Improve summary error message for invalid setting updates ([#4792](https://github.com/opensearch-project/OpenSearch/pull/4792))
+- Change InternalSignificantTerms to sum shard-level superset counts only in final reduce ([#8735](https://github.com/opensearch-project/OpenSearch/pull/8735))
```

Comment:
Are you targeting 2.10 for this change? If so, this needs to go in the [Unreleased 2.x] section.

Reply:
I honestly don't have a strong opinion about which section. We should not backport to 2.9 though. Only critical fixes or security fixes would go into a 2.9 patch release. A bug in an experimental feature would definitely not rise to that level.

Reply:
Got it. I moved it to the `Changed` section under unreleased 2.x i nthe most recent revision, thanks!